### PR TITLE
- Update chart.timeseries

### PR DESCRIPTION
**Pull request by zenithzhou**

### CODE REVIEW

**Comment by braverock** [comment]
travis build is still failing as of today's commits.

**Comment by erolbicero** [comment]
Build fails for me.  I had to delete the blank space between 'Imports:' and 'Depends:' for it to build/install.

File: DESCRIPTION [hunk 1]
```diff
@@ -28,7 +28,8 @@ Description: Collection of econometric functions for
 Imports:
     methods,
     quadprog,
-    zoo
+    zoo,
+
```
Comment (erolbicero) [inline DESCRIPTION]: This space caused problems for me.

File: R/chart.TimeSeries.R [hunk 2]
```diff
@@ -146,169 +146,139 @@
 #'                  event.color = "red", lwd = 2)
 #' 
 #' @export
-chart.TimeSeries <-
```
Comment (erolbicero) [inline R/chart.TimeSeries.R]: Running this fails for me: chart.TimeSeries(edhec, plot_engine = "plotly")

With the following error: 
'Error in layout(., title = main) : unused argument (title = main)'

File: R/chart.TimeSeries.R [hunk 3]
```diff
@@ -146,169 +146,139 @@
...
-	    minute = { date.format = "%H:%M"},
-	    hourly = {date.format = "%d %H"},
-	    daily = {if (yr_eq) date.format = "%b %d" else date.format = "%Y-%m-%d"},
-	    weekly = {if (yr_eq) date.format = "%b %d" else date.format = "%Y-%m-%d"},
-	    monthly = {if (yr_eq) date.format = "%b" else date.format = "%b %y"},
-	    quarterly = {if (yr_eq) date.format = "%b" else date.format = "%b %y"},
-	    yearly = {date.format = "%Y"}
-	)
+# New Function with multi-engine option
+chart.TimeSeries <-
```
Comment (erolbicero) [inline R/chart.TimeSeries.R]: Running this fails for me: chart.TimeSeries(edhec, plot_engine = "plotly")

With the following error:
'Error in layout(., title = main) : unused argument (title = main)'

Comment (zenithzhou) [reply R/chart.TimeSeries.R]: Updates are made for pass by dots, does the issue still exist?

**Comment by zenithzhou** [comment]
Move to a new pull request

**Pull request closed without merging**